packages/block-editor/src/components/block-mover/index.js

```diff
@@ -39,6 +40,9 @@ function BlockMover( {
 		return null;
 	}
 
+	const label =
+		clientIds.length === 1 ? __( 'Drag Block' ) : __( 'Drag Blocks' );
```

Comment:
Should use [`_n`](https://github.com/WordPress/gutenberg/blob/e7299d8347d7fc71b459ec5185c772066b1acd5b/packages/i18n/README.md#_n) for singular/plural situations:

```js
_n( 'Drag block', 'Drag blocks', clientIds.length )
```

(Also, prefer "sentence-case" style: _Drag Block_ -> _Drag block_)

Reply:
Thanks! I wasn't aware of it.  - Follow up PR here: https://github.com/WordPress/gutenberg/pull/25614

Also changed the remove option, as that was my hint to use capital `B` :)